### PR TITLE
BZ#1919804 remove Auto Pinning feature from docs - no longer supported

### PR DESCRIPTION
**Pull request by emarcusRH**
Signed-off-by: emarcusRH <emarcus@redhat.com>

Fixes issue # | \[BZ1919804](Bugzilla_URL)   for reference only - this bug closed long ago...

Changes proposed in this pull request:

-  Auto Pinning feature was removed - removing from documentation
-  Adding CPU Pinning Policy options - definitions

PREVIEWS:
https://ovirt.github.io/ovirt-site/previews/3085/documentation/virtual_machine_management_guide/index.html#Configuring_virtual_numa

https://ovirt.github.io/ovirt-site/previews/3085/documentation/virtual_machine_management_guide/index.html#Virtual_Machine_Resource_Allocation_settings_explained

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/main/CONTRIBUTING.md): (please @emarcusRH)
This pull request needs review by: (please @mention the reviewer if relevant)

### CODE REVIEW

**Review by liranr23 — COMMENTED** [review]
Can we have a pointer to the resource allocation Resize and Pin NUMA? In my opinion it is relevant both to the high performance and NUMA configuration.

The feature shouldn't be removed but changed. Now it can be done under the resource allocation, using the CPU pinning policy when selecting Resize and Pin NUMA.

**Review by sandrobonazzola — APPROVED** [review]

**Review by liranr23 — CHANGES_REQUESTED** [review]
> Can we have a pointer to the resource allocation Resize and Pin NUMA? In my opinion it is relevant both to the high performance and NUMA configuration.
> 
> The feature shouldn't be removed but changed. Now it can be done under the resource allocation, using the CPU pinning policy when selecting Resize and Pin NUMA.

^ same review

**Review by sandrobonazzola — DISMISSED** [review]
Please address @liranr23 comment

File: source/documentation/virtual_machine_management_guide/topics/Virtual_Machine_Resource_Allocation_settings_explained.adoc [hunk 1]
```diff
@@ -32,9 +32,14 @@ a|Allows users to set the level of CPU resources a virtual machine can demand re
 
 * *Custom* - A custom level of CPU shares defined by the user.
 | No.
+| |*CPU Pinning Policy*
+a|
 
+* `None` - runs without any CPU pinning.
 
-|
+* `Resize and Pin NUMA` - resizes the virtual CPU and NUMA topology of the virtual machine according to the Host, and pins them to the Host resources.
```
Comment (liranr23) [inline source/documentation/virtual_machine_management_guide/topics/Virtual_Machine_Resource_Allocation_settings_explained.adoc:40]: there are more policies here, dedicated, isolate threads and manual (to use the manual pinning described below).

File: source/documentation/virtual_machine_management_guide/topics/Configuring_virtual_NUMA.adoc [hunk 2]
```diff
@@ -22,13 +22,13 @@ If you define *NUMA Pinning*, the default migration mode is *Allow manual migrat
 . Click *NUMA Pinning*.
 . In the *NUMA Topology* window, click and drag virtual NUMA nodes from the box on the right to host NUMA nodes on the left as required, and click btn:[OK].
 . Select *Strict*, *Preferred*, or *Interleave* from the *Tune Mode* drop-down list in each NUMA node. If the selected mode is *Preferred*, the *NUMA Node Count* must be set to `1`.
-. You can also set the NUMA pinning policy automatically by selecting an *Auto Pinning Policy* from the drop-down list:
+.  You can also set the NUMA pinning policy automatically by selecting *Resize and Pin NUMA* from the CPU Pinning Polcy drop-down list under the *CPU Allocation* settings in the *Resource Allocation* tab:
```
Comment (liranr23) [inline source/documentation/virtual_machine_management_guide/topics/Configuring_virtual_NUMA.adoc:25]: extra space

File: source/documentation/virtual_machine_management_guide/topics/Virtual_Machine_Resource_Allocation_settings_explained.adoc (hunk 1 above)
Comment (emarcusRH) [reply source/documentation/virtual_machine_management_guide/topics/Virtual_Machine_Resource_Allocation_settings_explained.adoc:40]: I see those additional options, but they look like they are disabled - greyed out. 
Can I list them all as they they show up in the drop-down list?

**Comment by emarcusRH** [comment]
> Please address @liranr23 comment
done

File: source/documentation/virtual_machine_management_guide/topics/Configuring_virtual_NUMA.adoc [hunk 3]
```diff
@@ -22,13 +22,16 @@ If you define *NUMA Pinning*, the default migration mode is *Allow manual migrat
...
+. You can also set the NUMA pinning policy automatically by selecting *Resize and Pin NUMA* from the CPU Pinning Polcy drop-down list under the *CPU Allocation* settings in the *Resource Allocation* tab:
 +
-* *None* - makes no changes to the virtual machine.
-//* *Pin* - uses the existing CPU topology that has been set, in order to configure CPU pinning and NUMA pinning.
-* *Resize and Pin* - maximizes the CPU topology and generates the CPU pinning and NUMA pinning configurations.
+* `None` - runs without any CPU pinning.
+* `Manual` - available only when the virtual machine is pinned to a Host.
+* `Resize and Pin NUMA` - resizes the virtual CPU and NUMA topology of the virtual machine according to the Host, and pins them to the Host resources.
+* `Dedicated` - the selected CPU pinning policy is available for cluster compatibility level 4.7 or later. If the virtual machine has NUMA enabled, all nodes must be unpinned.
+* `Isolate Threads` - the selected CPU pinning policy is available for cluster compatibility level 4.7 or later. If the virtual machine has NUMA enabled, all nodes must be unpinned.
```
Comment (liranr23) [inline source/documentation/virtual_machine_management_guide/topics/Configuring_virtual_NUMA.adoc:31]: Worth adding what it actually does to the limitations? It also described in the UI when opening the drop-down list.
Manual - Runs manually specified virtual CPU on a specific physical CPU on a specific host.
Dedicated - Exclusively pin virtual CPUs to host physical CPUs.
Isolate Threads - Exclusively pin virtual CPUs to host physical CPUs. Each virtual CPU gets a physical core.

Without dropping the limitations above (host pinning, cluster level).

Unless it shouldn't be here and only the limitations are.

File: source/documentation/virtual_machine_management_guide/topics/Virtual_Machine_Resource_Allocation_settings_explained.adoc [hunk 4]
```diff
@@ -32,9 +32,16 @@ a|Allows users to set the level of CPU resources a virtual machine can demand re
...
 * *Custom* - A custom level of CPU shares defined by the user.
 | No.
+| |*CPU Pinning Policy*
+a|
 
+* `None` - runs without any CPU pinning.
+* `Manual` - available only when the virtual machine is pinned to a Host.
+* `Resize and Pin NUMA` - resizes the virtual CPU and NUMA topology of the virtual machine according to the Host, and pins them to the Host resources.
+* `Dedicated` - the selected CPU pinning policy is available for cluster compatibility level 4.7 or later. If the virtual machine has NUMA enabled, all nodes must be unpinned.
+* `Isolate Threads` - the selected CPU pinning policy is available for cluster compatibility level 4.7 or later. If the virtual machine has NUMA enabled, all nodes must be unpinned.
```
Comment (liranr23) [inline source/documentation/virtual_machine_management_guide/topics/Virtual_Machine_Resource_Allocation_settings_explained.adoc:42]: same as above

**Review by liranr23 — CHANGES_REQUESTED** [review]

**Review by liranr23 — APPROVED** [review]

**Review by sandrobonazzola — APPROVED** [review]

**Review by RichardHoch — APPROVED** [review]

**Pull request merged**